browser/src/StructuralVariantList/structuralVariantTableColumns.tsx

```diff
@@ -102,6 +102,16 @@ const structuralVariantTableColumns = [
     shouldShowInContext: (context: Context) => context.chrom !== 'Y',
   },
 
+  {
+    key: 'hemizygote_count',
+    heading: 'Number of Hemizygotes',
+    contextNotes: 'Not shown when viewing Y chromosome',
+    minWidth: 100,
+    compareFunction: makeNumericCompareFunction('ac_hemi'),
+    render: (variant: any) => renderAlleleCountCell(variant, 'ac_hemi'),
+    shouldShowInContext: (context: Context) => context.chrom !== 'Y',
```

Comment:
These attributes for the context and whether or not to show in the context appear to be duplicated from the above column (homozygote count) when I think they should be different.

For the short variants, this column has a note that it only appears for the sex chromosomes, and logic to achieve this. 

From `variantTableColumns.tsx`:

```
{
    key: 'hemizygote_count',
    heading: 'Number of Hemizygotes',
    description: 'Number of individuals hemizygous for alternate allele',
    contextNotes: 'Only shown when viewing X or Y chromosomes',
    grow: 0,
    minWidth: 100,
    compareFunction: makeNumericCompareFunction('ac_hemi'),
    render: (variant: Variant) => renderAlleleCountCell(variant, 'ac_hemi'),
    shouldShowInContext: (context: any) => context.chrom === 'X' || context.chrom === 'Y',
  },
```

I think we should at least mimic this behavior to stay consistent, this may also be an opportunity to re-use this column definition, rather than duplicating it.

Reply:
I also think we should add this column to the default columns rendered for SVs, and let the individual column definition logic handle whether or not its rendered.

The default columns are defined in `StructuralVariants.tsx`, on line 49:

```
const DEFAULT_COLUMNS = [
  'source',
  'consequence',
  'class',
  'pos',
  'length',
  'ac',
  'an',
  'af',
  'homozygote_count'
]
```

We could add `hemizygote_count` to the above list to achieve this.

Reply:
@rileyhgrant it's tempting to me to go through these lists of column specifiers and look for more copypasta to extract, but in the absence of tests and presence of other things I have to do, I'm leaving that for another day.

Reply:
@rileyhgrant ...actually, the two definitions of `hemizygote_count` are similar but not compatible, I'm just going to leave them separate and fix up the new SV version and call it a day until such time as I have the wherewithal to really overhaul this code.